puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/source/foreman-installer.groovy

```diff
@@ -1,4 +1,7 @@
 def commit_hash = ''
+source_type = 'rake'
```

Comment:
For my understanding: what's the difference between using `def` and not using def?

Reply:
`def` scopes the variable and `no def` keeps it global. I've seen issues where script variables with def are not seen by all parts of a script.

Reply:
In the future I would like to look at defining these in the job yaml file so we can template it anyway.